src/inmanta/agent/forking_executor.py

```diff
@@ -542,13 +544,13 @@ def mp_worker_entrypoint(
     logging.captureWarnings(True)
 
     # Set up our own logger
-    logger = logging.getLogger(f"agent.executor.{name}")
+    logger = logging.getLogger(f"{LOGGER_NAME_EXECUTOR}.{name}")
 
     # Load config
     inmanta.config.Config.load_config_from_dict(config)
 
     # Make sure logfire is configured correctly
-    tracing.configure_logfire("agent.executor")
+    tracing.configure_logfire("inmanta.executor")
```

Comment:
```suggestion
    tracing.configure_logfire(LOGGER_NAME_EXECUTOR)
```

Reply:
It is not about the logger name here (even if I'm not sure what it is)